docs(pie-docs): update engineer and foundation pages copy (#343)
justeattakeaway/pie: apps/pie-docs/src/foundations/typography/overview.md

---

## Our type scale

> Our type scale uses multiples of 4 for type sizes as well as line height, helping it align to our 4px vertical grid.

Reviewer: ```suggestion
Our type scale uses multiples of 4 for type sizes and line heights to align with our 4px vertical grid.
```

    caption: "The example above shows how a $heading-l looks on a narrow screen."
} %}

> **Only our heading and subheading typographic styles are responsive.** Our body styles share the same values regardless of the size of the screen they’re being displayed in.

Reviewer: ```suggestion
**Only our heading and subheading typographic styles are responsive.** Our body styles are the same for all screens.
```

---

## Font weights

> Font weight refers to a value assigned to your font which determines how bold or light your text will seem. We use three types of font weight.

Reviewer: ```suggestion
Font weight refers to how bold or light your text will appear; the higher the font weight, the bolder the text. We use three types of font weight.
```

Reviewer 2: 👍

---

## Alignment
This property defines the horizontal alignment of the text. It includes three types of alignment.

### Left aligned

> This is the preferred way to align text within our products. Left-aligned text performs better for readers because it helps users to easily identify the start of a new line.

Reviewer: <img src="https://user-images.githubusercontent.com/26894168/226684543-306a3ce7-e85a-4dae-bc1e-d6c52581ea70.png" width="300px" />

```suggestion
This is the preferred way to align text within our products because it makes it easier to identify the start of a new line (for left-to-right languages).
```

> This is the preferred way to align text within our products. Left-aligned text performs better for readers because it helps users to easily identify the start of a new line.

Reviewer: <img src="https://user-images.githubusercontent.com/26894168/226684543-306a3ce7-e85a-4dae-bc1e-d6c52581ea70.png" width="300px" />

```suggestion
This is the preferred way to align text within our products because it makes it easier to identify the start of a new line (for left-to-right languages).
```

Reviewer 2: 👍

---

## Line length

> Setting the right line length is key to the readability of the text in our products. With that in mind, a good practice is to keep the length between 80 and 100 characters per line, with a minimum of 60.

Reviewer 2: 👍

---

## Paragraph spacing

> It represents the amount of white spacing between two paragraphs of your text. Keeping consistent paragraph spacing measures throughout your products will help the user understand the hierarchy of the information and will make your texts easier to read. We use three measures.

Reviewer: ```suggestion
This is the space between two successive paragraphs of text. Keeping consistent paragraph spacing throughout your products will help the user understand the hierarchy of the information and will be easier to read. We use three measures.
```

Reviewer 2: 👍

### Paragraph Spacing 1

> This is the most commonly-used paragraph spacing measure. It uses **16px** between paragraphs.

Reviewer: ```suggestion
This is the most common paragraph spacing measure. It uses **16px** between paragraphs.
```

Reviewer 2: 👍
